samples/snippets/src/test/java/com/m/examples/bigtable/HelloWorldTest.java

```diff
@@ -12,10 +12,12 @@
  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
  * See the License for the specific language governing permissions and
  * limitations under the License.
-*/
+ */
 
 package com.m.examples.bigtable;
 
+import static junit.framework.TestCase.assertNull;
```

Comment:
i think `import static junit.framework.TestCase.assertNull;` would be `import static org.junit.Assert.assertNull;`

Reply:
Good catch! Thanks.